code/game/machinery/computer/fluff.dm

```diff
@@ -4,12 +4,45 @@
 #define DENY_TOO_OLD		1
 
 /obj/machinery/computer/fluff
+
+	var/window_data = "" //HTML to display to users
+	var/window_width  = 300
+	var/window_height = 300
+	var/allow_silicons = TRUE
```

Comment:
i understand why this exists but now that cyborgs use access i pout every time i see computers that use it

Reply:
You could make admins able to set the access needed to use the console but by default cyborgs have all station access. I'm not sure there's much that can be done, especially if you want to restrict AI's from using it(since AIs still ignore access).